homeassistant/components/husqvarna_automower/strings.json

```diff
@@ -253,5 +253,27 @@
         "name": "Avoid {stay_out_zone}"
       }
     }
+  },
+  "services": {
+    "park_for": {
+      "name": "Park for",
+      "description": "Parks the mower for a given time. The schedule will be overriden during that time.",
+      "fields": {
+        "duration": {
+          "name": "Duration",
+          "description": "How long the override will last in minutes."
+        }
+      }
+    },
+    "start_for": {
+      "name": "Start for",
```

Comment:
"Start for" doesn't sound good. Maybe "Mow for"?
Also, consider a common service which allows overriding the schedule for a certain duration, where the mode is set to mow or park.

Reply:
I changed it